src/view_floating.cpp

```diff
@@ -58,9 +59,9 @@ void View::DrawControlElement(DeviceContext *dc, ControlElement *element, Measur
     assert(measure);
     assert(element);
 
-    // For dir, dynam, and harm, we do not consider the @tstamp2 for rendering
+    // For dir, dynam, fermata, and harm, we do not consider the @tstamp2 for rendering
     if (element->HasInterface(INTERFACE_TIME_SPANNING) && (element->Is() != DIR) && (element->Is() != DYNAM)
-        && (element->Is() != HARM)) {
+        && (element->Is() != FERMATA) && (element->Is() != HARM)) {
```

Comment:
I am not sure this is necessary because a fermata has no TIME_SPANNING interface anyway